aws/resolve_customer/resolve_customer/entitlements.py

```diff
@@ -69,38 +69,41 @@ def __init__(self, customer_id: str, product_code: str):
             )
             log_error(self.error)
 
+    def get_toplevel_product_code(self) -> str:
+        toplevel_product = ''
+        if self.entitlements:
+            entitlements = self.entitlements.get('Entitlements')
+            if entitlements:
+                toplevel_product = entitlements[0].get('ProductCode')
+        return toplevel_product
+
     def get_entitlements(self) -> List[dict]:
-        entitlements = []
+        result_entitlements: List = []
         if self.entitlements:
-            for entitelement in self.entitlements['Entitlements']:
-                entitlements.append(
+            entitlements = self.entitlements.get('Entitlements') or []
+            for entitelement in entitlements:
```

Comment:
```suggestion
            for entitlement in entitlements:
```
probably, a typo

Reply:
Uh, yes but a consistent typo, I used the wrong name in all places :) Thanks good catch and fixed